addon/services/sl-behavior.js

```diff
@@ -35,120 +35,70 @@ export default Ember.Service.extend({
-
+    isAble( activity, resource, possible ) {
         Ember.assert(
-            'services/behavior.getBehaviorGroup() expects parameter to be an Object',
-            'instance' === providerType ||
-            'object' === providerType
+            'services/behavior.isAble() expects at least two parameters to be provided',
+            activity && resource
         );
 
-        Ember.assert(
-            'services/behavior.getBehaviorGroup() `provider.behaviorGroup` must be a String or Function',
-            'function' === behaviorGroupType ||
-            behaviorGroupTypeIsString
-        );
-
-        let behaviorGroup = ( behaviorGroupTypeIsString ) ?
-            Ember.get( provider, 'behaviorGroup' ) :
-            provider.behaviorGroup();
-
-        return behaviorGroup;
-    },
+        const behaviors = this.getWithDefault( 'behaviors', null );
 
-    /**
-     * Determine whether the desired behavior is able to be performed
-     *
-     * If `provider` is a string:
-     *     `provider` and `behavior` values are compared to data from `this.behaviors` to make determination.
-     *
-     * If `provider` is an object:
-     *     Object provides string value to be used as `provider` value
-     *     `provider` value is first compared to data from `this.behaviors` to make determination
-     *     If present then it further looks to `behaviors` hash on object for result of property with `provider` value
-     *
-     * @function
-     * @param {String} behavior
-     * @param {String|Object} provider
-     * @throws {ember.assert} If no parameters provided, first parameter is not a string, or second parameter is not a
-     *     string or object
-     * @returns {Boolean}
-     */
-    isAble( behavior, provider ) {
         Ember.assert(
-            'services/behavior.isAble() expects two parameters to be provided',
-            behavior && provider
+            'services/behavior.isAble() expects `activity` parameter to be a String',
+            'string' === Ember.typeOf( activity )
         );
 
-        let behaviors = this.getWithDefault( 'behaviors', null );
-        let providerType = Ember.typeOf( provider );
-        let providerIsObject = providerType === 'instance' || providerType === 'object';
-
         Ember.assert(
-            'services/behavior.isAble() expects `behavior` parameter to be a String',
-            'string' === Ember.typeOf( behavior )
+            'services/behavior.isAble() expects `resource` parameter to be a String',
+            'string' === Ember.typeOf( resource )
         );
 
         Ember.assert(
-            'services/behavior.isAble() expects `provider` parameter to be either a String or Object',
-            'string' === providerType || providerIsObject
+            'services/behavior.isAble() expects `possible` parameter to be a Boolean or undefined',
+            'boolean' === Ember.typeOf( possible ) || 'undefined' === Ember.typeOf( possible )
```

Comment:
We have been breaking these out onto multiple lines, such as:

```
Ember.assert(
    'services/behavior.isAble() expects `possible` parameter to be a Boolean or undefined',
    'boolean' === Ember.typeOf( possible ) ||
    'undefined' === Ember.typeOf( possible )
    );
```
though don't recall if the style guide mentions that specifically regarding multiple assertion conditions.